SunEngine.Tests/SunEngine.Admin/Managers/CacheSettingsManagerTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using SunEngine.Admin.Managers;
+using SunEngine.Core.Cache.Services;
+using SunEngine.Core.Configuration.Options;
+using SunEngine.Core.DataBase;
+using SunEngine.Core.Models;
+using Xunit;
+
+namespace SunEngine.Tests.SunEngine.Admin.Managers
+{
+    public class CacheSettingsManagerTests
+    {
+        private CacheSettingsManager cacheSettingsManager { get; set; }
+        private DataBaseConnection dbConnection { get; set; }
+
+        public CacheSettingsManagerTests()
+        {
+            dbConnection = DefaultInit.GetTestDataBaseConnection();
+            cacheSettingsManager = InitialiseDefaultSettingsManager();
+        }
+
+        private CacheSettingsManager InitialiseDefaultSettingsManager()
+        {
+            var serviceCollection = new ServiceCollection();
+            serviceCollection.AddOptions<IOptions<CacheOptions>>();
+            serviceCollection.AddMemoryCache();
+            var serviceProvider = serviceCollection.BuildServiceProvider();
+
+            var categoriesCache = new CategoriesCache(DefaultInit.GetTestDataBaseFactory());
+            var cacheOptions = serviceProvider.GetRequiredService<IOptions<CacheOptions>>();
+            var contentCache = new CategoryContentCache(categoriesCache, serviceProvider.GetRequiredService<IMemoryCache>(), serviceProvider.GetRequiredService<IOptions<CacheOptions>>());
+            return new CacheSettingsManager(cacheOptions, contentCache, dbConnection);
+        }
+
+        [Fact]
+        public async void ShouldUpdateCachePolicy()
+        {
+            using (var transaction = dbConnection.BeginTransaction())
+            {
+                var cacheSettings = dbConnection.CacheSettings.First();
+                var before = cacheSettings.InvalidateCacheTime;
```

Comment:
I would suggest using linebreaks to highlight logic separation.
1. Retrieve and save before value
2. Update DB with new value
3. Validate result.